packages/fxa-content-server/app/scripts/templates/sign_in_password.mustache

```diff
@@ -19,7 +19,6 @@
     <div class="success"></div>
 
     {{{ userCardHTML }}}
-    {{{ accountSuggestionHTML }}}
```

Comment:
Was this missed in #12153 or is was this removed by mistake? 👀

Reply:
@LZoog Yea it was 😅